1-getting-started.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `1-getting-started.ipynb` Cell 89 Line 37_
```markdown
5. Using the Run Button:
* In the top-right corner of the editor, there’s a green play button (Run Python File). Click it to execute the file.

#### How to Make `.ipynb` markdown-like Cells in VS Code

VS Code supports Notebook-like functionality using "Python Interactive" or Jupyter Notebook integration.

1. Option 1: Use Code Cells in a .py File
```
That's cool, I didn't know you could do that!